src/main/webapp/assets/modules/current-job/salary-review.con.js

```diff
@@ -115,38 +115,31 @@ techlooper.controller("salaryReviewController", function ($scope, $rootScope, js
       }
       delete $scope.error[modelName];
       var inputValue = $scope.$eval(modelName);
-      var notHasValue = ($.type(inputValue) === "array") && inputValue.length > 0;
-      notHasValue = notHasValue || ('' + inputValue).length > 0;
-      notHasValue || ($scope.error[modelName] = sprintf("%s %s", $(input).attr("name"), translate.isRequired.toLowerCase()));
+      var notHasValue = ($.type(inputValue) === "array") && (inputValue.length === 0);
+      notHasValue = notHasValue || (inputValue === null);
+      notHasValue = notHasValue || (inputValue.length <= 0);
+      notHasValue && ($scope.error[modelName] = sprintf("%s %s", $(input).attr("name"), translate.isRequired.toLowerCase()));
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.